docs/integrations/data-ingestion/clickpipes/aws-privatelink.md

```diff
@@ -0,0 +1,158 @@
+import cp_rpe_step3 from '@site/static/images/integrations/data-ingestion/clickpipes/cp_rpe_step3.png';
+import cp_rpe_settings0 from '@site/static/images/integrations/data-ingestion/clickpipes/cp_rpe_settings0.png';
+import cp_rpe_settings1 from '@site/static/images/integrations/data-ingestion/clickpipes/cp_rpe_settings1.png';
+import Image from '@theme/IdealImage';
+
+# AWS PrivateLink for ClickPipes
+
+You can use [AWS PrivateLink](https://aws.amazon.com/privatelink/) to establish secure connectivity between VPCs,
+AWS services, your on-premises systems, and ClickHouse Cloud without exposing traffic to the public Internet.
+
+This document outlines the ClickPipes reverse private endpoint functionality
+that allows setting up an AWS PrivateLink VPC endpoint.
+
+## Supported AWS PrivateLink endpoint types {#aws-privatelink-endpoint-types}
+
+ClickPipes reverse private endpoint can be configured with one of the following AWS PrivateLink approaches:
+
+- [VPC resource](https://docs.aws.amazon.com/vpc/latest/privatelink/privatelink-access-resources.html)
+- [MSK multi-VPC connectivity for MSK ClickPipe](https://docs.aws.amazon.com/msk/latest/developerguide/aws-access-mult-vpc.html)
+- [VPC endpoint service](https://docs.aws.amazon.com/vpc/latest/privatelink/privatelink-share-your-services.html)
+
+Follow the links above for detailed instructions on how to set up the respective AWS PrivateLink shares.
+
+### VPC resource {#vpc-resource}
+
+Your VPC resources can be accessed in ClickPipes using PrivateLink.
+Resource configuration can be targeted with a specific host or RDS cluster ARN.
+Cross-region is not supported.
+
+See a [getting started](https://docs.aws.amazon.com/vpc/latest/privatelink/resource-configuration.html) guide for more details.
+
+It's a preferred choice for Postgres CDC ingesting data from RDS cluster.
+
+### MSK multi-VPC connectivity {#msk-multi-vpc}
+
+The MSK multi-VPC is a built-in feature of AWS MSK that allows you to connect multiple VPCs to a single MSK cluster.
+Private DNS support is out of the box and does not require any additional configuration.
+Cross-region is not supported.
+
+It is a recommended option for ClickPipes for MSK.
+See a [getting started](https://docs.aws.amazon.com/msk/latest/developerguide/mvpc-getting-started.html) guide for more details.
+
+### VPC endpoint service {#vpc-endpoint-service}
+
+VPC service is another approach to share your data source with ClickPipes.
+It requires setting up a NLB (Network Load Balancer) in front of your data source
+and configuring the VPC endpoint service to use the NLB.
+
+VPC endpoint service can be [configured with a private DNS](https://docs.aws.amazon.com/vpc/latest/privatelink/manage-dns-names.html),
+that will be accessible in a ClickPipes VPC.
+
+Cross-region is supported.
+
+It's a preferred choice for:
+- any on premise Kafka setup that requires private DNS support
+- cross-region connectivity for Postgres CDC
```

Comment:
nit: should we should just say `cross-region connectivity` since it applies to both Kafka and Postgres CDC